test/feature/initializing_controllers.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Initializing controllers
+
```

Comment:
Please add a description of the end-user value here, since that isn't obvious just from the spec body. Something like

```cucumber
Feature: Dependency Injection into controllers

As a developer using Exprestive to instantiate my controllers for me
I want to be able to define dependencies to be provided to my controllers
So that I can use my non-Exprestive application objects from within controllers without having to resort to global variables.
```

In general, the `Feature` line provides a super short summary of the feature. The end-user value describes WHY a feature exists. The scenarios describe how the feature works.